theano/compile/function_module.py

```diff
@@ -1475,7 +1475,8 @@ def __init__(self, inputs, outputs,
                     optimizer_profile = self.optimize_graph_with_cache(
                         optimizer, inputs, outputs)
                 else:
-                    optimizer_profile = optimizer(fgraph)
+                    with theano.configparser.change_flags(compute_test_value="off"):
```

Comment:
There is already the option `theano.config.compute_test_value_opt` that governs this behaviour, and its default is `"off"`. Why override an option here that has been explicitly set?